crates/web/frontend/src/components/header/star-count.tsx

```diff
@@ -0,0 +1,50 @@
+import { cn } from "@/lib/utils";
+import { Github, Star } from "lucide-react";
+import { type MouseEvent, useCallback, useEffect, useState } from "react";
+import styles from "./header.module.css";
+
+interface Props {
+	className?: string;
+}
+
+const StarCount = ({ className }: Props) => {
+	const [stars, setStars] = useState<number>();
+	const [clicked, setClicked] = useState<boolean>(false);
+
+	const handleClick = useCallback((e: MouseEvent<HTMLAnchorElement>) => {
+		setClicked(true);
+		setTimeout(() => {
+			window.open("https://github.com/jorgehermo9/gq", "_blank");
+			setClicked(false);
+		}, 0);
+		e.preventDefault();
+	}, []);
+
+	useEffect(() => {
+		fetch("https://api.github.com/repos/jorgehermo9/gq")
+			.then((res) => res.json())
+			.then((data) => setStars(data.stargazers_count));
+	}, []);
+
+	return (
+		<a
+			href="https://github.com/jorgehermo9/gq"
+			onClick={handleClick}
```

Comment:
is it necessary a handleClick ig you already have an anchor with href?

Reply:
if you need to handle the setClick, maybe you can stop preventing the default event on that function so the redirect is done by the native anchor tag instead of you calling the window.open